pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -104,6 +111,25 @@ func (r *rbac) createResources(ctx context.Context) error {
 		return nil
 	}
 
+	exist, err := checkIfInstallerSetExist(ctx, r.operatorClientSet, r.version, r.tektonConfig, componentName)
```

Comment:
can we do a direct TektonInstallerSets.Get() using the name `rbac-resources` to check the expected InstallerSet exists.